nautobot/ipam/models.py

```diff
@@ -524,6 +525,9 @@ class Prefix(PrimaryModel, StatusModel):
         "is_pool",
         "description",
     ]
+    dynamic_group_filter_fields = {
```

Comment:
Comment: Why is this necessary? Explain similarly to how it is done on `VirtualMachine`.